extism/go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/extism/cli/extism
 
 go 1.20
 
-// replace github.com/extism/cli => ../
+replace github.com/extism/cli => ../
```

Comment:
need to figure out why I changed this a while back (we shouldn't use the published version in our own module, right?

Reply:
i can't remember why either, but this would be very helpful! we should uncomment this line for extism-dev too if we decide it's okay to keep it